File: decompiled/sources/com/hjq/permissions/OnPermissionCallback.java
package com.hjq.permissions;

import java.util.List;

public interface OnPermissionCallback {

    /* renamed from: com.hjq.permissions.OnPermissionCallback$-CC  reason: invalid class name */
    public final /* synthetic */ class CC {
        public static void $default$onDenied(OnPermissionCallback onPermissionCallback, List list, boolean z) {
        }
    }

    void onDenied(List<String> list, boolean z);

    void onGranted(List<String> list, boolean z);
}

File: decompiled/sources/com/hjq/permissions/OnPermissionCallback.kt
package com.hjq.permissions

interface OnPermissionCallback {
    fun onDenied(deniedPermissions: List<String>, doNotAskAgain: Boolean) {}

    fun onGranted(grantedPermissions: List<String>, allGranted: Boolean)

    object CC {
        @JvmStatic
        fun `$default$onDenied`(
            onPermissionCallback: OnPermissionCallback,
            deniedPermissions: List<String>,
            doNotAskAgain: Boolean
        ) {
        }
    }
}
